extensions/wikia/FacebookClient/scripts/preferences.js

```diff
@@ -24,7 +24,14 @@
 			$disconnectButton = $('.fb-disconnect');
 			$connectLink = $('.sso-login-facebook');
 
-			$.loadFacebookAPI(bindEvents);
+			$.loadFacebookAPI()
+				.done(function () {
+					$('.fb-loaded').removeClass('hidden');
```

Comment:
In this case, the `$connectLink` is already cached, so we should be querying the DOM for it